src/LiveComponent/assets/src/Backend/BackendResponse.ts

```diff
@@ -13,4 +14,12 @@ export default class {
 
         return this.body;
     }
+
+    async getLiveUrl(): Promise<string | null> {
+        if (undefined === this.liveUrl) {
+            this.liveUrl = await this.response.headers.get('X-Live-Url');
```

Comment:
Can by sync, `Headers.get()` is not sync:
```suggestion
    getLiveUrl(): string | null {
        if (undefined === this.liveUrl) {
            this.liveUrl = this.response.headers.get('X-Live-Url');
```